config/indexer.go

```diff
@@ -69,7 +69,7 @@ func NewIndexer() Indexer {
 		GCTimeLimit:         Duration(5 * time.Minute),
 		ShutdownTimeout:     0,
 		ValueStoreDir:       "valuestore",
-		ValueStoreType:      "sth",
+		ValueStoreType:      "pebble",
```

Comment:
This is probably the most significant change the PR.

I recommend rewording the title PR to include it for the benefit of someone looking through commit history and reading titles of it, as well as making the release notes point out what's important when autogenerated from PR merges.